dist/main.js.map

```diff
@@ -1 +1 @@
AAG,gBAAgB,CAAC,aAAa;AACpD,QAAQ,CAAC,KAAK,EAAE,WAAW,EAAE,IAAI,EAAE,QAAQ,CAAC,CAAC,CAAC;AAC9C,MAAM,OAAO,CAAC,IAAI,CAAC,OAAO,IAAI,KAAK,CAAC,CAAC;AACrC,KAAK;AACL,IAAI,YAAY,CAAC,GAAG,CAAC,MAAM,EAAE,MAAM,GAAG,OAAO,GAAG,OAAO,CAAC,CAAC,CAAC,CAAC,CAAC;AAC5D,GAAG;AACH;AACA,EAAE,oBAAoB,CAAC,CAAC,OAAO,EAAE,QAAQ,EAAE,KAAK,EAAE,QAAQ,CAAC,EAAE;AAC7D,IAAI,MAAM,YAAY,GAAG,aAAa,CAAC,GAAG,CAAC,QAAQ,CAAC,CAAC;AACrD,IAAI,MAAM,OAAO,GAAG,YAAY,IAAI,YAAY,CAAC,aAAa;AAC9D,MAAM,CAAC,KAAK,EAAE,WAAW,EAAE,IAAI,EAAE,QAAQ,CAAC,CAAC,CAAC;AAC5C,IAAI,GAAG,OAAO,EAAE;AAChB,MAAM,OAAO,CAAC,IAAI,CAAC,OAAO,CAAC,CAAC;AAC5B,MAAM,OAAO,IAAI,CAAC;AAClB,KAAK;AACL,GAAG;AACH;AACA,EAAE,MAAM,eAAe,CAAC,CAAC,OAAO,EAAE,YAAY,EAAE,KAAK,CAAC,EAAE;AACxD;AACA,IAAI,MAAM,QAAQ,GAAG,EAAE,CAAC;AACxB,IAAI,IAAI,MAAM,GAAG,IAAI,KAAK,EAAE;AAC5B,MAAM,MAAM,QAAQ,GAAG,IAAI,GAAG,EAAE,CAAC;AACjC,MAAM,QAAQ,CAAC,IAAI,CAAC,QAAQ,CAAC,CAAC;AAC9B,MAAM,MAAM,IAAI,CAAC,UAAU,CAAC,CAAC,GAAG,EAAE,YAAY,EAAE,QAAQ,EAAE,YAAY,CAAC,CAAC,CAAC;AACzE,KAAK;AACL,IAAI,OAAO,CAAC,IAAI,CAAC,QAAQ,CAAC,CAAC;AAC3B,GAAG;AACH;AACA,EAAE,MAAM,gBAAgB,CAAC,CAAC,OAAO,EAAE,YAAY,EAAE,KAAK,CAAC,EAAE;AACzD;AACA,IAAI,MAAM,YAAY,GAAG,IAAI,GAAG,EAAE,CAAC;AACnC,IAAI,OAAO,CAAC,IAAI,CAAC,YAAY,CAAC,CAAC;AAC/B,IAAI,MAAM,IAAI,CAAC,UAAU,CAAC,CAAC,GAAG,EAAE,KAAK,EAAE,YAAY,EAAE,YAAY,CAAC,CAAC,CAAC;AACpE,GAAG;AACH;AACA,EAAE,cAAc,CAAC,CAAC,OAAO,EAAE,YAAY,EAAE,QAAQ,CAAC,EAAE;AACpD,IAAI,MAAM,CAAC,MAAM,EAAE,MAAM,CAAC,GAAG,QAAQ,CAAC;AACtC,IAAI,YAAY,CAAC,GAAG,CAAC,MAAM,EAAE,MAAM,GAAG,OAAO,GAAG,OAAO,CAAC,CAAC,CAAC,CAAC,CAAC;AAC5D,GAAG;AACH,CAAC;AACD;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;;AC/MA;AACA;AACA;AAIA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACO,eAAe,MAAM,CAAC;AAC7B,EAAE,cAAc,EAAE,cAAc,EAAE,aAAa,GAAG,IAAI,GAAG,EAAE;AAC3D,EAAE,eAAe,GAAG,CAAC,EAAE,QAAQ;AAC/B,CAAC,GAAG,EAAE,EAAE;AACR,EAAE,GAAG,EAAE,eAAe,KAAK,CAAC,IAAI,eAAe,KAAK,CAAC,CAAC,EAAE;AACxD,IAAI,MAAM,IAAI,SAAS;AACvB,MAAM,wDAAwD;AAC9D,MAAM,iCAAiC,CAAC,CAAC;AACzC,GAAG;AACH;AACA;AACA;AACA;AACA;AACA;AACA,EAAE,MAAM,MAAM,GAAG,IAAI,UAAU,CAAC,CAAC,IAAI,EAAE,IAAI,EAAE,eAAe,CAAC,CAAC,CAAC;AAC/D,EAAE,IAAI,MAAM,CAAC;AACb;AACA,EAAE,GAAG,eAAe,KAAK,CAAC,EAAE;AAC5B;AACA,IAAI,MAAM,GAAGA,gBAAK,CAAC,MAAM,CAAC,cAAc,CAAC,CAAC;AAC1C,GAAG,MAAM;AACT;AACA,IAAI,MAAM,UAAU,GAAG,IAAI,UAAU,CAAC,CAAC,cAAc,EAAE,aAAa,CAAC,CAAC,CAAC;AACvE,IAAI,MAAM,GAAG,MAAM,UAAU,CAAC,QAAQ,CAAC,CAAC,cAAc,EAAE,QAAQ,CAAC,CAAC,CAAC;AACnE,GAAG;AACH;AACA;AACA,EAAE,MAAM,MAAM,GAAG,MAAM,CAAC,MAAM,GAAG,MAAM,CAAC,MAAM,CAAC;AAC/C,EAAE,MAAM,KAAK,GAAG,IAAI,UAAU,CAAC,MAAM,CAAC,CAAC;AACvC,EAAE,KAAK,CAAC,GAAG,CAAC,MAAM,CAAC,CAAC;AACpB,EAAE,KAAK,CAAC,GAAG,CAAC,MAAM,EAAE,MAAM,CAAC,MAAM,CAAC,CAAC;AACnC;AACA,EAAE,GAAG,QAAQ,EAAE;AACf,IAAI,QAAQ,CAAC,4BAA4B,CAAC,CAAC;AAC3C,IAAI,QAAQ,CAACC,YAAO,CAAC,KAAK,EAAE,CAAC,KAAK,EAAE,IAAI,EAAE,MAAM,EAAE,IAAI,CAAC,CAAC,CAAC,CAAC;AAC1D,GAAG;AACH;AACA,EAAE,OAAO,KAAK,CAAC;AACf,CAAC;AACD;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;AACA;;;;;"}
```

Comment:
Is this meant to be included here?

Reply:
@kdenhartog Yes, because we are pulling this git repository as NPM dependency ([link to code](https://github.com/mattrglobal/opencore/blob/master/packages/core-cborld/package.json#L40)), we need to also commit the built artifacts (including this source map file).